src/rpc/blockchain.cpp

```diff
@@ -168,7 +168,7 @@ UniValue blockToDeltasJSON(const CBlock& block, const CBlockIndex* blockindex)
                 if (IsValidDestination(dest)) {
                     delta.pushKV("address", keyIO.EncodeDestination(dest));
                 }
-                delta.pushKV("satoshis", -1 * spentInfo.satoshis);
+                delta.pushKV("zatoshis", -1 * spentInfo.zatoshis);
```

Comment:
Revert this and other renames involving outputs of RPC calls (including everything in this file).